articles/resources/relnotes.md

```diff
@@ -17,6 +17,25 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.12.20092501
+
+*Release date: September 29th, 2020*
+
+This release contains the following:
+
+- Performance improvements for simulator: better application of gate fusion decisions; improved parallelization on Linux system; added intelligent scheduling of gate execution; bug fixes; overall speed improvement of ~5x (even faster on larger problems).
```

Comment:
- Introduction of [Quantum Intermediate Representation (QIR)](https://github.com/microsoft/qsharp-language/tree/main/Specifications/QIR#quantum-intermediate-representation-qir) intended as a common format across different front- and backends; see our [blog](https://devblogs.microsoft.com/qsharp/introducing-quantum-intermediate-representation-qir)
- Launch of our new Q# language [repo](https://github.com/microsoft/qsharp-language) containing also the full [Q# documentation](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language#q-language)